MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.PopupBox.xaml

```diff
@@ -118,11 +118,25 @@
                             </wpf:Card>
                         </controlzEx:PopupEx>
                     </Grid>
+                    <ControlTemplate.Triggers>
+                        <Trigger Property="IsEnabled" Value="false">
+                            <Setter Property="Opacity" Value="0.23"/>
+                        </Trigger>
+                    </ControlTemplate.Triggers>
                 </ControlTemplate>
             </Setter.Value>
         </Setter>
     </Style>
 
+    <Style x:Key="MaterialDesignToolPopupBox" BasedOn="{StaticResource MaterialDesignPopupBox}" TargetType="wpf:PopupBox">
+        <Setter Property="Foreground" Value="#616161" />
+        <Setter Property="Padding" Value="4"/>
+    </Style>
+
+    <Style x:Key="MaterialDesignToolForegroundPopupBox" BasedOn="{StaticResource MaterialDesignToolPopupBox}" TargetType="wpf:PopupBox">
```

Comment:
Yes, i've got a new card using several of the new styles. I'll merge from master and and the card to this PR.